### ISSUE TITLE
Availability of an Alternate Logo

### ISSUE THREAD

**Issue by zoffixznet** [perl6/user-experience#5]
For those unaware, the [original Camelia logo](http://perl6.org/camelia-logo.png) has taken a lot of critique and the main takeaway can be the following: the logo is not very serious and some people would be uncomfortable to present materials that have it to, say, someone I'm going to dub as "boring corporate drone" types.

For Perl 6 to spread as a language of choice in corporations, I think Perl 6 developers would be the original push. They will be showing "something" to their bosses. And some of those bosses might not be too kin on the happy, colourful logo.

I'm perfectly happy with the current logo and by no means advocate we change it, but I propose we create an alternate variant of it that is more boring, more muted, more appealing to the "boring corporate drones."

The easiest take would be to just mute Camelia's colours. I also see @MadcapJake has tried their hand at creating a style similar to some Python logo: https://twitter.com/MadcapJake/status/687859550397976578 . I think with some additional work, it can be one of the alternate logos.

These alternate logos designed to appeal to a specific segment of people would be used on marketing materials created for that segment.

**Comment by ShaneKilkelly** [comment]
There should definitely be a viable alternative logo, even just because not every context suits a screaming colourful logo.

**Comment by AlexDaniel** [comment]
There is also a [black and white](https://www.flickr.com/photos/jessrobinson/15967316925/) variant (it was done multiple times).

Honestly, I don't see any problem. I've never felt that there is something wrong with it. The ideas behind the current logo are pretty straightforward too.

[This](http://static.independent.co.uk/s3fs-public/styles/story_large/public/thumbnails/image/2015/10/01/15/deathmetalposter.jpg) was also once mentioned in one of those IRC discussions. The thing is: it is OK to stand out.

I'd say that some people should just deal with it. Perl 6 is good enough by itself, there's no need to make a “professional” logo to attract people. Any professional who thinks that the current logo is unprofessional should think why his tool choice depends on the logo.

Another important note is that alternative logo will not make the main logo go away (unless that's what you mean). “They will be showing "something" to their bosses” – yeah, and these bosses will go to perl6.org and see the main logo. I'm not sure what this alternative logo is solving in this case.

That being said, it's totally fine if the logo is black&white sometimes. So if you want camelia to be sad and dead, sure, just use it. But there's no need to call it alternative logo.

**Comment by zoffixznet** [comment]
> I've never felt that there is something wrong with it. The ideas behind the current logo are pretty straightforward too.  [...] The thing is: it is OK to stand out.

Sure, you may have a unique style of dress that is [quirky and stands out](https://therunwayjournal.files.wordpress.com/2013/11/qbx_lfw_aw_catalog_12.jpg) and all your friends know the ideas behind your style. But were you to dress like that for a job interview, you'd likely be on a losing side: the person potentially picking you has to make a quick decision to select you out of a dozen of options. Any red flag can disqualify you, simply because there are other options and learning more about you **requires precious time**. No one cares about the ideas behind your style unless they already chose you and want to know more.

> I'd say that some people should just deal with it. [...] Any professional who thinks that the current logo is unprofessional should think why his tool choice depends on the logo.

Unfortunately, another way to phrase it is: "we don't want to reach any audience that doesn't like us already." If a monkey would walk into a car dealership, it'd be shown brochures with monkeys driving those cars. We can do the same with brochures for our language. Especially since we have a designer with appropriate tools on staff.

The complaints voiced about the logo are coming from real people. Expecting them to "just deal with it" and somehow magically to progress further with Perl 6 is simply unrealistic when they're hundreds of other languages to choose from.

**Comment by ShaneKilkelly** [comment]
I'm with @zoffixznet on this. There's no point stubbornly dismissing concerns over the logo, as it's something that can harm perl6's adoption in a very real way.

Having an alternate logo for those who want something a bit less free-jazz is a good idea.

**Comment by MadcapJake** [comment]
@AlexDaniel some people *do* see a problem.  As I've said before in other places, I like the logo—I especially like what it stands for.  However, as @ShaneKilkelly stated above, a super colorful logo doesn't fit every context. Other programming websites might want to include a perl6 tag or category and having a logo that fits more there design will be useful and encouraging to others.  Ultimately I think we do live in an age where presentation is more important than we'd all like to admit.

Let's try and remember, this whole repository is about growing adoption and gaining new Perl 6 users, not everyone is going to have the same principled views on logo design (in terms of preferring principles over aesthetics).  We should try and keep the "just deal with it" sentiment out of the conversation.

**Comment by AlexDaniel** [comment]
Sure this all makes sense, but most people complaining about the logo also don't like the butterfly itself, not just the way it is colored. 

Anyway, what do you guys think about black&white version? It seems like it fits “Other programming websites might want to include a perl6 tag”, “something a bit less free-jazz” and “dress like that for a job interview, you'd likely be on a losing side” kinds of arguments.

In other words, why not just use black&white version for all things where beautiful camelia is not wanted?

**Comment by jberger** [comment]
Of all the "marketing problems" of p6, the logo is the one I care about the least. After all golang's gopher isn't very professional looking. That said the gopher is really just a character. I'm sure they have an official image of it, but I see that character in all sorts of poses and color schemes and everyone knows that it is a pseudo-valid alternative logo without having to propose it.

**Comment by uvtc** [comment]
> For Perl 6 to spread as a language of choice in corporations, I think Perl 6 developers would be the original push. They will be showing "something" to their bosses.

My impression (please correct me if I'm wrong) is that, with Perl 5, it generally wasn't "presented it to the boss" per se ... Perl simply began being used because it was too terribly useful *not* to use it. By the time anyone "presented" it, it was already running the shop. See also <https://xkcd.com/224/>.

If Perl 6 is useful and valuable, then (I think) the logo shouldn't matter. The people who utilize Perl 6 won't be put off by the logo, and the people who benefit from the solutions created using Perl 6 will only see the logo later, if at all.

Personally, I don't see a need for an alternate, more corp-friendly logo. For those concerned though, would it make sense to have a more boring corp-friendly Perl 6 homepage (using an alt logo) that could live at perl6.*com* (as opposed to the regular site at perl6.org)?

@zoffixznet you mentioned the Python logo. Interesting to compare the old and new ones http://cobweb.cs.uga.edu/~maria/classes/4500-Spring-2012/figs/pythonLogoSmall.png wrt to the Perl 6 logo.

**Comment by polettix** [comment]
I hope you will pardon me for the huge email, I found a lot of things
around and I try to give a few alternatives.

Another important note is that alternative logo will not make the main logo
> go away (unless that's what you mean). “They will be showing "something" to
> their bosses” – yeah, and these bosses will go to perl6.org and see the
> main logo. I'm not sure what this alternative logo is solving in this case.
>
This is an interesting observation that qualifies as a fact (i.e.
non-debatable, at least today). At some point, you will probably have to
deal with Camelia anyway.


Some past experience about this issue:

The /about/ page: *"Camelia as the logo for Perl 6 and perl6.org
<http://perl6.org> is not negotiable."*.

camelia.txt <https://github.com/perl6/mu/blob/master/misc/camelia.txt>:*
"Those of you who think the current design does not reflect good taste are
entitled to your opinion.  We will certainly allow you to change your mind
later as you grow younger.  :)"*

An interesting quote from Larry Wall: "*Up until now, the Python community
has done a much better job of getting into the lower levels of education
than we have. We’d like to do something in that space too, and that’s
partly why we have the butterfly logo, because it’s going to be appealing
to seven year old girls!"*


Now, some alternatives.

As a person showing Perl 6 to "that" boss, you can e.g. avoid putting the
logo, or do a fake logo yourself, or use the textual variant below. You
just have to remember what already stated above: most probably, you will
have to deal with Camelia at some time.

There is already an officially approved alternative to the Camelia logo
(again, from camelia.txt
<https://github.com/perl6/mu/blob/master/misc/camelia.txt>), and it's this:

 »ö«

Put a box around it and presto! you have a (arguably) corporate-safe logo.

Another alternative would be telling "that" boss that they're going to use
some *specific* implementation of Perl6, which is probably what matters
most to the management. Rakudo Perl 6 has a logo that hardly qualifies as
childish and "not safe for corporate management". camelia.txt
<https://github.com/perl6/mu/blob/master/misc/camelia.txt> is pretty clear
to this regard: *"In particular, various implementations and distributions
are free to use their own logos and/or mascots."*.


A different logo? Let's see...

I think that it's perfectly fine to have a "Perl 6 fan art" page, so that
it's clear that it's nothing official but it's anyway easy to find and use.
There Is More Than One Way To Disguise It, if you want to use a different
icon go for it, but don't complain if nobody endorses it officially. Caveat
Emptor.

There's also space for a *really* different logo. camelia.txt
<https://github.com/perl6/mu/blob/master/misc/camelia.txt> file also
contains this excerpt from 2009:







*"So let me summarize the requirements into a meta-requirement:    The new
logo must make Larry at least as happy as Camelia does.That is the extent
to which my mind is still open...  :-)    Larry"*

I don't see anything blocking anyone from proposing something new, in the
hope that it will make Larry as happy as Camelia does. Hence, there might
be a contest for a logo replacement, with proposals made public (subject to
light moderation approval and with proper fine prints).

Something like the fan art page, but asking Larry to choose the best (so,
it must include Camelia anyway!). Who knows, there MIGHT be a logo designed
by a real 7-years old girl that MIGHT make Larry at least as happy as
Camelia does.

**Comment by zoffixznet** [comment]
> I think that it's perfectly fine to have a "Perl 6 fan art" page

:+1: on having a "fan art" page. @MadcapJake already started a [repo with logo variations](https://github.com/MadcapJake/metamorphosis). GitHub itself has a whole ton of user-made Octocat versions: https://octodex.github.com/

**Comment by raiph** [comment]
![Perl 6 Hints @perl6hints twitter account image](https://pbs.twimg.com/profile_images/684910689421516801/6evedH-w_400x400.png)

**Comment by xfix** [comment]
@zoffixznet: Just to make it clear, Octodex only contains official Octocat variations, as described on https://octodex.github.com/faq.html. If you check who made those, they are only made by GitHub Staff, so it isn't exactly an example.

**Comment by bbkr** [comment]
https://github.com/perl6/perl6.org/issues/3 - there was issue opened to change logo back in 2011:)

Camel was perfect logo for Perl. It was symbol of endurance, ability to carry heavy stuff for long period of time, reliability.

Butterfly symbolises transformation, but It's not good long-term logo. 10 years from now no one will remember Apocalypses, decision to break backward compatibility, "a-ha" moments (jnthns chart :P), frustration when half ecosystem was broken because of spec changes, and so on.

I think that for corp-drones we should simply redesign camel.
Butterfly will be childish - no matter what color scheme it has.

**Comment by MadcapJake** [comment]
@bbkr I think what you said in that thread is really a critical point that I'd like to copy here:

> Language has to be presented as a **trustworthy** tool - so people are willing to bet their data, reputation and money on it.

**Comment by jberger** [comment]
> Language has to be presented as a trustworthy tool - so people are willing to bet their data, reputation and money on it.

And if you base that decision on a logo, you're doing it wrong.

**Comment by MadcapJake** [comment]
The reason the saying "You can't judge a book by its cover" exists is because most people judge books by their covers.

**Comment by ShaneKilkelly** [comment]
We can insist the logo is not a problem all we want, but that's not going to change how the logo, and by extension perl6 itself, is perceived by the wider programmer community.

**Comment by AlexDaniel** [comment]
> The reason the saying "You can't judge a book by its cover" exists is because most people judge books by their covers.

Personally I don't mind if some people stay away from Perl 6. This may sound a bit too rough, but there are definitely some benefits of having such built-in filter.

Anyway, **here is an idea**:
What if instead of having colorful, round camelia we make a variant that is more “industrial”? Robotmelia! Sharp edges, gray colors, wires and cogs sticking out, perhaps even hydraulic cylinders. But don't overdo it, we probably don't want to end up with a terminator. Other than that it could be badass.

Also, a lot of kids like robots, so… perhaps this will satisfy both parties to some extent. Oh, and by the way, I think that just changing the colors (like it was done in a green logo) wont get enough support from people who really like the current logo. It just makes things worse without making it significantly “better” for people who don't like the butterfly itself.

**Comment by AlexDaniel** [comment]
> Butterfly symbolises transformation, but It's not good long-term logo.

I think that it is appropriate because you can change the grammar.

**Comment by polettix** [comment]
An executive's mind is often focused on their own protection. *What if I accept to use this weird, new language my people is making such a fuss? Will my manager ridicule me and will this backfire on my career?*

Making life easier to anyone willing to give it a corporate try should be at least considered.

> Camelia is intended primarily to represent Perl 6, The Language,
> not any other aspect of Perl 6 culture, corporate or otherwise.
> In particular, various implementations and distributions are free to
> use their own logos and/or mascots.

(from [camelia.txt](https://raw.githubusercontent.com/perl6/mu/master/misc/camelia.txt)). Again, why not push for "adopting" Rakudo (an implementation) and use its logo? Or create a logo to represent "Corporate Perl 6"?

Re Camel, please remember that this symbol associated to Perl is a trademark by O'Reilly ([see here](http://archive.oreilly.com/pub/a/oreilly/perl/usage/)). Any use requires their permission.

**Comment by cjepeway** [comment]
Holy cow, @jberger is all over it.

Go's gopher mascot is goofy as all get-out...but real people and real enterprises are using go in real important places.  See https://github.com/golang/go/wiki/GoUsers#companies-currently-using-go-throughout-the-world

That's an existence proof, then, that you don't need a straight-laced logo to get your programming language adopted pretty widely.

And that "goofy" thing I just wrote?  That's wrong.  The gopher's not goofy...it's itself, nothing else.

Let's not even get into Java's Duke, OK?

I say leave Camelia alone.  She's cute, she's fun, and she's cool.

**Comment by nige123** [comment]
> I say leave Camelia alone. She's cute, she's fun, and she's cool.

Hear hear!

She's distinctive - an important feature in any logo.

She references a 'camel' in her name - so she "runs deep" in Perl culture -
unlike a camel though she doesn't smell or spit and doesn't infringe the
trademark owned by O'Reilly.

For me she symbolises O(fun) - the propensity to find 'flow' while coding -
a seriously productive state of mind which even the stuffy suits will
understand is good for business [1].
Nige

1.
https://perl6advent.wordpress.com/2015/12/20/perl-6-christmas-have-an-appropriate-amount-of-fun/

**Comment by zaucker** [comment]
While not disagreeing with your statement to make everybodies life easier, I
don't see how a camel would be perceived completely differently. At least
in German "du Kamel" ("you camel") is actually an insult as camels are
perceived as being stupid.

And, btw, snakes are not usually perceived as very nice creatures either.

It is not entirely clear to me if this topic really fits into this mailing
list or repository.  So far I fail to see how it connects to "user
experience".  And, for sure, not a single posting in this thread actually
reported any experience with the logo being a reason for Perl6 not being
used anywhere.  To me the whole discussion seems to be based on peoples'
fantasies.

Of course, one could see this discussion fitting under "When encountering a
Perl 6 user who seems to be frustrated by any part of the Perl 6 experience,
it is suggested they get directed to open an Issue on this repository, so
that the Perl 6 Community can address it." But even for this kind of
interpretation I'd suggest the frustrated user sharing some real life
experience with us instead of invoking "sayings".

BTW, I am not a great fan of the logo. But there seem to be a lot more
pressing issues to be addressed than this. And, somehow I can see the
personality behind Camelia as a real plus. Just follow the Perl6 irc (log)
for a while and enjoy the friendliness of the community (towards each other,
towards new comers, towards trolls showing up every once in a while).
Perhaps Camelia has something to do with this? If so, I'd strongly vote for
keeping her around (of course, this is also just a fantasy).

Cheers,
Fritz

On Sun, 17 Jan 2016, Flavio Poletti wrote:

> An executive's mind is often focused on their own protection.  *What if I
> accept to use this weird, new language my people is making such a fuss? 
> Will my manager ridicule me and will this backfire on my career?*
>
> Making life easier to anyone willing to give it a corporate try should be
> at least considered.
>
>> Camelia is intended primarily to represent Perl 6, The Language,
>> not any other aspect of Perl 6 culture, corporate or otherwise.
>> In particular, various implementations and distributions are free to
>> use their own logos and/or mascots.

-- 
Oetiker+Partner AG              tel: +41 62 775 9903 (direct)
Fritz Zaucker                        +41 62 775 9900 (switch board)
Aarweg 15                            +41 79 675 0630 (mobile)
CH-4600 Olten                   fax: +41 62 775 9905
Schweiz                         web: www.oetiker.ch

**Comment by polettix** [comment]
> not a single posting in this thread actually reported any experience with the logo being a reason for Perl6 not being used anywhere. To me the whole discussion seems to be based on peoples' fantasies.

Indeed, this thread started from a speculation that is supported only by whining elsewhere:

> [...] some people would be uncomfortable to present materials that have it to, say, someone I'm going to dub as "boring corporate drone" types.

How "comfortable" one is to use Perl 6 is somehow related to user experience. And yes, this consideration doesn't make this thread any more "scientific".

My view is that there is nothing pressing for a *change* to Camelia, actually, as the representative for the *language*. I don't like the graphics personally, but I like the idea behind it (the "seven year old girl" stuff discussed by LW) and I wouldn't be surprised if it helped creating the right spirit in the community.

Everyone that is uncomfortable with it in some *formal* context should probably turn to the implementation's logo, as the implementation is what they would be analyzing and evaluating anyway. Today this would probably be [Rakudo](http://rakudo.org/), whose logo has the symmetry of a butterfly and *ehr*... nothing else.

There can also be intermediate steps in which a more boring logo can be designed to represent use of Perl 6 in boring contexts, which is allowed by [the rules](https://raw.githubusercontent.com/perl6/mu/master/misc/camelia.txt) apparently.

**Comment by pmichaud** [comment]
On Mon, Jan 18, 2016 at 12:05:32AM -0800, Flavio Poletti wrote:
> Everyone that is uncomfortable with it in some *formal* context should 
> probably turn to the implementation's logo, as the implementation is 
> what they would be analyzing and evaluating anyway. Today this would 
> probably be [Rakudo](http://rakudo.org/), whose logo has the 
> symmetry of a butterfly and *ehr*... nothing else.

Hear hear.  I've said for years that Rakudo can (and should) have a logo
that is separate from Camelia.  If you want precedence for this, think
Linux (Tux the penguin) versus Ubuntu (circle of friends), RedHat (a fedora),
Debian (a swirl), SUSE (a veiled chameleon), etc.  

Beyond that, I think that Linux is a prime example of how a cartoonish
system mascot (Tux the penguin) isn't really a barrier to adoption in 
"corporate" contexts.

Those who are trying to blur the lines between "Perl 6" and "Rakudo" 
seem to be (perhaps unintentionally) embarking on building a Perl 6
cathedral, when we're aiming for something more bazaar-like.  I 
understand that it feels a bit simpler and more familiar when one can 
use a single term like "Perl 6" to refer synonymously to both a language 
and an implementation, but Perl has been down that road before 
(with Perls 1-5) and doing so carries its own set of baggage.

Personally, I think some of the monochrome variations of Camelia 
mentioned earlier in this thread would work just fine in more 
corporate contexts.  Camelia doesn't have to always be full-color,
as she can change her wing colors at will, including to versions
involving no color scheme whatsoever (e.g., completely black).

Lastly, in response to Pawel Pabian's comment of "Butterfly symbolises 
transformation, but it's not good long-term logo.  10 years from now 
no one will remember ...":  I think this really misses an important 
point about Perl 6.  Part of the reason Perl 6 has needed a long 
gestational period is precisely because it's intended to be a language 
that can will continually transform decades into the future.  The 
transformations we think of in Perl 6 aren't merely about what has
been changed from Perl 5, but but also all of the future changes that 
remain to be discovered and we have no clue about.

True long-term stability in a language will only come if it's able to
continually transform and adapt to a changing world.  That is the
transformation that Camelia is intended to evoke and symbolize.

Pm

**Comment by zoffixznet** [comment]
It seems this discussion has significantly derailed from the original topic.

Replacing Camelia was never a proposal. The topic was making available a toned-down logo to rebuke any complaints that the official logo is too bright and not corporate-like. It's quick and simple to do and doesn't require several pages of philosophical musings.

I'm going to close this, at least for now, until, if ever, some concepts are available to look at.

**Comment by JohnLonginotto** [comment]
Python user here. I saw Perl6's logo on someone's blog and thought "wow, what a crappy logo for you blog - the rest of the blog looks nice, but this logo seems totally out of place."

Then I realised that this is the official Perl6 logo, and i must admit, i laughed. Then I read this discussion and I laughed even harder. 101 ways to do the same thing in Perl, and now you guys want 101 logos, haha. 

After reading the comments, i'm a little more sympathetic to the logo. You want to attract young people and stand out. That's noble - i guess. But I worry that Perl is becoming the hipster language that's more about being different than it is about practicality. Practically, this logo is not ideal. It's going to put off more people than the camel. Also the reaction from some of 'deal with it' is totally bizarre to me. Personally, i'd go with something colourful and geometric. Like, er... 

https://pe56d.s3.amazonaws.com/p19ij9oa41j3qs5idfq1cklm7o9.jpg

or 

http://image.shutterstock.com/z/stock-photo-geometric-butterfly-with-many-brilliant-triangles-swallowtail-122158450.jpg

although i was actually thinking more line-art than vonrois, but you get the idea. The design is mature, the colours say "and it's fun". Then you corporate types can grayscale it if you wish. I'd still call it Camilia, because that's a cool pun, but i'd lose the big eyes and smiley face. Even kids don't think that's cool..

Anyway, just a brutally honest opinion from someone who isn't at all invested in the Perl community and was considering it but the logo put me off. First impressions matter, and anyone who says don't judge a book by it's cover is being overly idealistic. Can you imagine the firing you'd get after suggesting Perl, the boss says they don't like it, logo looks dumb, but he conceeds, perl gets used, setbacks incurred as they always are, boss says "i told you so!", and the rest is history..

**Comment by raiph** [comment]
TL;DR While imo this issue is rightly closed, and I think responding to Longinotto's comment here is almost like feeding an undercover troll disguised as a grinch, forgive me as I post, for the record, about a mini drama from yesterday involving a reddit user with the user account u/Longinotto, and then some relevant advice from Larry Wall.

----

First, about u/Longinotto:

I posted a link to Ken Clark's advent post as [a reddit in /r/Bioinformatics](https://www.reddit.com/r/bioinformatics/comments/5g2jxi/bioinformatics_with_perl_6/) yesterday. It garnered 59 comments, mostly the usual condemnation of even considering Perl.

User u/Longinotto posted the first comment.

They've since deleted all their reddit posts so I can't show you what they wrote and can't prove that the Longinotto that posted here in this user-experience thread is the same person. But I think the latter pretty likely and I think that what they wrote on reddit might matter (slightly, at least for this issue thread here), hence this comment I'm writing.

In response to Longinotto, Ken Clark wrote in the thread (under his hunkamunka reddit nick) "I see from your comment history that you simply hate Perl's syntax".

I didn't look at Longinotto's history but I can confirm that u/Longinotto's initial post in the above linked thread was a troll: a half dozen lines that went something like: "Oh, this is going to be good"; two and a half lines of code taken from Ken's post that u/Longinotto non-sensically ran together as a single non code line (presumably because it radically increased the line noise effect); commentary literally consisting of slight variations on "Hahahahahahaha"; and, finally, some substance written as "Hey the 90s called, they wanted you to parse HTML or structured data with a regex".

----

Finally, I'll quote Larry from [an advent blog post from 2010](https://perl6advent.wordpress.com/2010/12/24/day-24-yule-the-ancient-troll-tide-carol/):

Camelia has turned out to be a most useful cultural hack, that tells us with a fair degree of certainty who the grinches are who want to steal Christmas. Every community has to deal with an influx of potentially poisonous people, and having an obvious target like Camelia to complain about induces such people to wave a flag reading: “Hey, I’m a troll. Hug me.”

And that’s another cultural hack in the Perl 6 community. We believe in hugging trolls. Up to a point.

...

So when we say we hug trolls, what we really mean in technical terms is that we don’t pay much attention to your position when you join us, however odious that position might be. Instead, we look at the first and second derivatives of your position. To give us time to differentiate, we typically perform a bit of verbal aikido to let you express some of your deeper yearnings while you think you are merely tormenting us.

...

In short, there are grinches, but some grinches will repent. We want to give them the chance. Sometimes this involves singing when the grinch steals all the presents.

But some of you grinches will never repent. Did we mention that Camelia has a 3-meter wingspan? And that she likes to suck the brains from unrepentant grinches? Not only that, but Camelia’s larval form was a camel, so she can spit too. You really, really don’t want to have Camelia sucking your brain and then spitting.

But most people don’t need to have their brains either sucked or spat. Often they just need them washed. Once people catch on to the meta-philosophy of Perl, they usually find the search for technical and cultural convergence to be a Great Adventure, and much more fun than simply making people unhappy, which is too easy.
